Open Judge System/OJS.Services/Data/Problems/ProblemsDataService.cs

```diff
@@ -13,28 +13,57 @@ public class ProblemsDataService : IProblemsDataService
         public ProblemsDataService(IEfDeletableEntityRepository<Problem> problems) =>
             this.problems = problems;
 
-        public IQueryable<Problem> GetByIdQuery(int id) =>
-            this.problems.All().Where(p => p.Id == id);
+        public Problem GetById(int id) => this.problems.GetById(id);
 
         public Problem GetWithProblemGroupById(int id) =>
             this.problems.All().Include(p => p.ProblemGroup).FirstOrDefault(p => p.Id == id);
 
         public Problem GetWithContestById(int id) =>
             this.problems.All().Include(p => p.ProblemGroup.Contest).FirstOrDefault(p => p.Id == id);
```

Comment:
Why don't you put these method calls on separate lines?

Reply:
because they are less than 120 characters long